packages/nodejs/src/span.ts

```diff
@@ -241,4 +243,19 @@ export class RootSpan extends BaseSpan {
       this._ref = span.createRootSpan(namespace)
     }
   }
+
+  /*
+   * Adds a given `Error` object to the current `RootSpan`.
+   */
+  public addError(error: Error): this {
```

Comment:
```suggestion
  public setError(error: Error): this {
```

We don't want to make `addError` public.

Reply:
This function must remain public, it's used from the Tracer and from the ScopeManager. Sometimes we want to call `RootSpan.setError()` from the outside when the Tracer is not available or needed.

Reply:
My comment was unclear. We don't want to make the logic of an `addError` public yet, adding multiple errors, because we don't support that yet. So we stay with `setError` for now.